src/v3/src/util/locUtil.ts

```diff
@@ -13,16 +13,41 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { loc as localize } from 'okta';
 
+import { TokenReplacementValue, TokenSearchValue } from '../types';
+
+/**
+ *
+ * @param key - i18n key used for the translation
+ * @param bundleName - Translation bundle to read properties from
+ * @param params - parameters used to interpolate tokens in the string
+ * @param {Record<TokenSearchValue, TokenReplacementValue>} tokenReplacement - Record that enables
+ * you to find and replace tokens embedded in the string.
+ * @returns translated string in the current language code
+ */
 export const loc = (
   key: string,
   bundleName?: string,
   params?: Array<string | number | boolean | unknown>,
+  tokenReplacement?: Record<TokenSearchValue, TokenReplacementValue>,
 ): string => {
-  const localizedText = localize(key, bundleName, params);
+  const localizedText: string = localize(key, bundleName, params);
+
+  if (typeof tokenReplacement !== 'undefined') {
+    let updatedText = localizedText;
```

Comment:
curious, does this work with nested tags? so like `text text <$1> text text <$1> text </$1> </$1>`

Reply:
It would not work with that :( , but I don't believe that is expected to work. With these tokens, if there are multiple open/closing tags in one string, the token value should change. 

i.e. 
```
hcaptcha.footer.label = This site is protected by hCaptcha and its <$1>Privacy Policy</$1> and <$2>Terms of Service</$2> apply.
```

So in this case, if we wanted to, we would just need to add `$2` to the TokenSearchValue type and it will work as expected.

Reply:
What if I wanted to format lets say:
`<a href="#" > <strong> here is a test string </strong> </a>`

do we support that?

Reply:
We _shouldn't_ support that 😅 adding HTML elements inside the translation string is sort of considered an anti-pattern and we actually are discussing how we can clean up these translations and limit how things can be styled

Reply:
ah I see, so is signin widget going for no HTML elements at all?

Because from what I'm aware of, we don't want HTML in the `properties` explicitly but it's ok to be using these `<$1>` notations. 

What I was going for is, if that was the theoretical string that we wanted to represent, would `<$1> <$2> here is a test string </$2> </$1>` work with the function you have built out if I fill in the proper arguments for `$1` and `$2`?

Reply:
Ah, then yes that is possible, I had to modify my type a bit so TS would not complain but here is the commit. Also added a unit test to show it working. 

https://github.com/okta/okta-signin-widget/pull/3088/commits/9e461921955d2bac6294891807f5cb78e79f30fa

Reply:
Epic, thank you 👍